Windows/Linux app service plans must be in different groups

mjrousos opened the pull request:
The current docs stated that Linux and Windows app service plans should be in different regions. Instead, it should say that they must be in different *resource groups*.

azurecla commented:
Hi __@Juliako__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (Juliako). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Juliako commented:
#sign-off

mjrousos committed:
Update app-service-linux-intro.md

The current docs stated that Linux and Windows app service plans should be in different regions. Instead, it should say that they must be in different *resource groups*.
